packs/aws/config.yaml

```diff
@@ -1,8 +1,8 @@
 ---
 setup:
-  region: ""
-  aws_access_key_id: ""
-  aws_secret_access_key: ""
+  region: "us-east-1"
```

Comment:
Not sure what the standard is here but was trying to go for something that worked for many people out of the box...   I can change if needed